test/util/rational.cpp

```diff
@@ -30,11 +30,11 @@ void test_coordinate_cast(std::string const& s, int expected_nom, int expected_d
 void test_numeric_cast()
 {
     const boost::rational<int> r1(3, 4);
-    BOOST_CHECK_CLOSE(bg::detail::numeric_cast<double>(r1), 0.75, 0.00001);
+    BOOST_CHECK_CLOSE(bg::util::numeric_cast<double>(r1), 0.75, 0.00001);
 
     const boost::rational<int> r2(10, 4);
-    BOOST_CHECK_CLOSE(bg::detail::numeric_cast<double>(r2), 2.5, 0.00001);
-    BOOST_CHECK_EQUAL(bg::detail::numeric_cast<int>(r2), 2);
+    BOOST_CHECK_CLOSE(bg::util::numeric_cast<double>(r2), 2.5, 0.00001);
+    BOOST_CHECK_EQUAL(bg::util::numeric_cast<int>(r2), 2);
```

Comment:
Oops. As soon as the Cmake PR is merged ,this won't happen anymore ;-)
Thanks for fixing.